airflow/api_connexion/endpoints/dag_endpoint.py

```diff
@@ -81,11 +81,13 @@ def get_dag_details(
     for key, value in dag.__dict__.items():
         if not key.startswith("_") and not hasattr(dag_model, key):
             setattr(dag_model, key, value)
+
     try:
         dag_detail_schema = DAGDetailSchema(only=fields) if fields else DAGDetailSchema()
     except ValueError as e:
         raise BadRequest("DAGDetailSchema init error", detail=str(e))
-    return dag_detail_schema.dump(dag_model)
+    serialized_dag_model = dag_detail_schema.dump(dag_model)
+    return serialized_dag_model
```

Comment:
```suggestion
    return dag_detail_schema.dump(dag_model)
```

serialized_dag_model means something specific, namely `SerializedDagModel`

which is not this

so let's avoid confusion

Reply:
Changed it to `dag_detail_schema`